src/main/java/core/basesyntax/strategy/operation/SupplyOperationHandler.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.strategy.operation;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.strategy.OperationHandler;
+
+public class SupplyOperationHandler implements OperationHandler {
+    @Override
+    public void accept(String fruit, String quantity) {
+        Storage.STORAGE.put(fruit, Storage.STORAGE.get(fruit) + Integer.parseInt(quantity));
```

Comment:
what if we do not have such fruit in storage yet?
add null checks for addition operations